### PR TITLE
Add loading indicator

### PR DESCRIPTION
**Pull request by fenichelar**

### CODE REVIEW

**Comment by fenichelar** [comment]
Ping

**Comment by mydea** [comment]
Just my perspective, I'm not sure if that is better than what we currently have (e.g. a loading spinner via CSS pseudo elements). The issue is that this will tear down the whole table with everything in it once it starts loading, and then re-build it completely, which I imagine might not be ideal performance wise. Also it might be somewhat jarring if loading times are very short.

File: addon/components/models-table.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ const defaultMessages = {
   'columns-hideAll': 'Hide All',
   'columns-restoreDefaults': 'Restore Defaults',
   tableSummary: 'Show %@ - %@ of %@',
+  loading: 'Loading...',
```
Comment (DimkaVardina) [inline addon/components/models-table.js]: It should be checked how the default texts could be replaced with custom ones in order to use translations

File: addon/templates/components/models-table.hbs [hunk 2]
```diff
@@ -58,4 +60,4 @@
   {{/if}}
   {{! table footer end }}
 </div>
-{{yield}}
+{{yield}}
```
Comment (DimkaVardina) [inline addon/templates/components/models-table.hbs]: New line

File: tests/dummy/app/controllers/examples/filtering-select-box-bool.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import generateContent from '../../utils/c';
+
+const {A} = Ember;
+
+export default Ember.Controller.extend({
+
+  title: 'Filtering with select-box (by boolean values)',
+  model: generateContent(10),
+  columns: A([
```
Comment (DimkaVardina) [inline tests/dummy/app/controllers/examples/filtering-select-box-bool.js]: Why is this statement needed?

**Pull request closed without merging**